pkg/collector/corechecks/snmp/internal/devicecheck/devicecheck.go

```diff
@@ -109,10 +110,12 @@ func (d *DeviceCheck) Run(collectionTime time.Time) error {
 	if checkErr != nil {
 		tags = append(tags, d.savedDynamicTags...)
 		d.sender.ServiceCheck(serviceCheckName, metrics.ServiceCheckCritical, tags, checkErr.Error())
+		d.sender.Gauge(deviceUpMetric, 0., tags)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Instead of a new metric, I'm wondering if we can add a new tag to existing https://github.com/DataDog/datadog-agent/blob/ddf59f0dfd86481963e6f2846eecdbfe8d44ea73/pkg/collector/corechecks/snmp/internal/devicecheck/devicecheck.go#L286 `snmp.device_monitored` metric ? tags like `connect_status:ok/critical` (we can use the same status as the service check for example)